src/app/dwg/qgsdwgimportdialog.cpp

```diff
@@ -443,24 +452,31 @@ void QgsDwgImportDialog::createGroup( QgsLayerTreeGroup *group, const QString &n
 
     l->setLabeling( new QgsVectorLayerSimpleLabeling( pls ) );
     l->setLabelsEnabled( true );
+    layers.append( l );
   }
 
-  l = layer( layerGroup, layerFilter, QStringLiteral( "points" ) );
+  l = createLayer( layerFilter, QStringLiteral( "points" ) );
   if ( l )
   {
     // FIXME: use PDMODE?
     l->setRenderer( new QgsNullSymbolRenderer() );
+    layers.append( l );
   }
 
-  if ( !cbExpandInserts->isChecked() )
+  return layers;
+}
+
+void QgsDwgImportDialog::createGroup( QgsLayerTreeGroup *group, const QString &name, const QStringList &layers, bool visible )
+{
+  QgsLayerTreeGroup *layerGroup = group->addGroup( name );
+  QgsDebugMsg( QStringLiteral( " %1" ).arg( name ) ) ;
+  Q_ASSERT( layerGroup );
+
+  const QList<QgsVectorLayer *> layersList = createLayers( layers );
+  for ( QgsVectorLayer *layer : layersList )
   {
-    l = layer( layerGroup, layerFilter, QStringLiteral( "inserts" ) );
-    if ( l && l->renderer() )
-    {
-      QgsSingleSymbolRenderer *ssr = dynamic_cast<QgsSingleSymbolRenderer *>( l->renderer() );
```

Comment:
Can you explain why this code is no longer required?

Reply:
It is a mistake, I re-added it.

Reply:
I added a test that checks that those awaited layers are effectively in the project after using the import dialog.
@nyalldawson could you please take another look? I would like to have this into 3.32